extensions/pi-authentik/AUTHENTIK_SETUP.md

```
@@ -0,0 +1,85 @@
+
+After authentik is configured:
+
+1. Add the `pi-authentik` config to Pi settings or run `/authentik-setup`
+2. Start Pi
+3. Run `/authentik-login`
+4. Complete login in the browser
+5. Pi stores the session in `pi-secret` and registers the provider
+
+## Troubleshooting
+
+### Redirect mismatch
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)